pymess/backend/push/onesignal.py

```diff
@@ -44,7 +44,9 @@ def publish_message(self, message):
             include_external_user_ids=(message.recipient,),
             contents={language: message.content for language in languages},
             headings={language: message.heading for language in languages},
-            data=message.extra_data,
+            data={
+                'redirectUrl': message.redirect_url
+            } if message.redirect_url else {},
```

Comment:
this is wrong

there you should join extra data with this value

Reply:
fixed